moped-database/views/project_list_view.sql

```diff
@@ -121,8 +121,24 @@ AS WITH project_person_list_lookup AS (
         AND ptags.is_deleted = false
         AND ptags.project_id = mp.project_id
       GROUP BY ptags.project_id) AS project_tags,
-    string_agg(contracts.contractor, ', ') AS contractors,
-    string_agg(contracts.contract_number, ', ') AS contract_numbers
+    ( -- get me all of the contractors added to a project
+      SELECT string_agg(contract.contractor, ', ' :: text) AS string_agg
+      FROM moped_proj_contract contract
+      WHERE 1 = 1
```

Comment:
i'm skeptical that we need these `1=1` in these various subqueries, but i forgot to test that out

Reply:
thanks @mddilley and @chiaberry for filling me in. good to know they are not cruft! ✅